sdl_android/src/androidTest/assets/xml/MOBILE_API_4.5.0.xml

```diff
@@ -1688,55 +1688,55 @@
         </param>
         <param name="latitudeDegrees" type="Float" minvalue="-90" maxvalue="90" mandatory="true">
         </param>
-        <param name="utcYear" type="Integer" minvalue="2010" maxvalue="2100" mandatory="true">
```

Comment:
This file should not have any changes to it, as this is not part of the 4.5 spec

Reply:
Would it be preferred for us to create a MOBILE_API_4.7.0.xml file with the API changes, add it to the project and reference it from: https://github.com/smartdevicelink/sdl_android/blob/develop/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/RPCConstructorsTests.java#L28
(leaving MOBILE_API_4.5.0.xml unchanged)

Reply:
I need to look into this and will get back to you soon